terraform/modules/paas/variables.tf

```diff
@@ -36,4 +36,12 @@ locals {
   worker_app_start_command = "bundle exec sidekiq -C config/sidekiq.yml"
   worker_app_name          = "register-worker-${var.app_environment}"
   logging_service_name     = "register-logit-${var.app_environment}"
+
+  service_gov_uk_host_names = {
+    qa      = ["qa"]
+    staging = ["staging"]
+    prod    = ["www", "www2"]
```

Comment:
you don't need `www2` and also array of hostnames

Reply:
ok.. i thought that was needed by PaaS..